scripts/core/registry.yml

```diff
@@ -367,9 +367,18 @@ etors:
 - name: COMMAND_BUFFER_FINALIZE_EXP
   desc: Enumerator for $xCommandBufferFinalizeExp
   value: '123'
+- name: COMMAND_BUFFER_APPEND_MEMBUFFER_WRITE_EXP
```

Comment:
Annoyingly this shouldn't happen - we should always allocate new registry entries from the largest value. Let me investigate what's going on here. :+1:

Reply:
I did that and it solves the issue. Thanks